ProcessMaker/modeler
# Fix sequence flow labels bug

Fixes #269.

**How to test:**
* Create an exclusive gateway and task, and connect them with a sequence flow.
* Edit the sequence flow expression; ensure it updates the shape.
* Ensure you can undo and redo the above change.
